src/compiler/parser.bi

```diff
@@ -315,7 +315,8 @@ declare function cSymbolType _
 		byref subtype as FBSYMBOL ptr, _
 		byref lgt as longint = 0, _
 		byref is_fixlenstr as integer = FALSE, _
-		byval options as FB_SYMBTYPEOPT = FB_SYMBTYPEOPT_DEFAULT _
+		byval options as FB_SYMBTYPEOPT = FB_SYMBTYPEOPT_DEFAULT, _
+		byval mode as INTEGER = 0 _                  
```

Comment:
I've noticed this in several files from your repos, that there is trailing space added at the end of the line.  Why?